src/chanakya/services/mcp_wrapper.py

```
@@ -0,0 +1,73 @@
+
+def forward_stdin(proc):
+    """Forward stdin lines to the subprocess until EOF."""
+    try:
+        while True:
+            line = sys.stdin.readline()
+            if not line:
+                break
+            proc.stdin.write(line)
+            proc.stdin.flush()
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)